src/Draco.Compiler/Internal/Syntax/ParseTreeFormatter.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Draco.Compiler.Api.Syntax;
+using Token = Draco.Compiler.Internal.Syntax.ParseTree.Token;
+
+namespace Draco.Compiler.Internal.Syntax;
+
+internal sealed record class ParseTreeFormatterSettings(string Indentation);
```

Comment:
Shouldn't this be something that could be in the public API? Since tools would consume this, this would be something that could be affected by editorconfig, ...